packages/@vue/cli-plugin-pwa/ui.js

```diff
@@ -58,7 +64,12 @@ module.exports = api => {
             message: 'org.vue.pwa.config.pwa.backgroundColor.message',
             description: 'org.vue.pwa.config.pwa.backgroundColor.description',
             default: '#000000',
-            value: data.manifest && data.manifest.background_color,
+            value:
+              (data.vue &&
+                data.vue.pwa &&
+                data.vue.pwa.manifestOptions &&
+                data.vue.pwa.manifestOptions.background_color) ||
+              (data.manifest && data.manifest.background_color),
```

Comment:
We really need optional chaining 😅

Reply:
👀Time to use TypeScript for every project!